examples/nats-basic-secure-connection/publisher.bal

```diff
@@ -0,0 +1,27 @@
+import ballerinax/nats;
+
+public function main() returns error? {
+    // To secure the client connection using TLS/SSL, the client needs to be configured with
+    // a certificate file of the server.
+    // The [`nats:SecureSocket`](https://docs.central.ballerina.io/ballerinax/nats/latest/records/SecureSocket)
+    // record provides th   e SSL-related configurations of the client.
+    nats:SecureSocket sslConfig = { cert: "../resource/path/to/public.crt" };
+
+    // To secure the client connections using basic authentication, provide the credentials
+    // with the [`nats:Credentials`](https://docs.central.ballerina.io/ballerinax/nats/latest/records/Credentials) record.
+    nats:Credentials credentials = { username: "ballerina",
```

Comment:
Shall we use `alice` and `alice@123` for username and password? It is not wrong to use something else, but this is what is used in common for security examples.